app/containers/Bookings/Details/QuickLook/index.js

```diff
@@ -719,6 +743,16 @@ class QuickLook extends Component {
                 />
               </div>
             )}
+            {isPathfinderUser && (
+              <div className="col-xs-12">
+                <h3 className="heading-medium">Pathfinder</h3>
+
+                <Pathfinder
+                  pathfinderUrl={pathfinderUrl && `${pathfinderUrl}/refer/offender/${offenderNo}`}
+                  isPathfinderUser={isPathfinderUser}
```

Comment:
why do you need this check here - you're already checking on line 746 anyway?

Reply:
Looks similar, but line 746 checks whether this is a Pathfinder user whilst 751 checks whether the pathfinder URL is set before using it.  The values isPathfinderUser and pathfinderUrl are both supplied into the Pathfinder function so that the unit test in Bookings/Details/QuickLook/tests/pathfinder.test.js can try different scenarios when conditionally rendering the link. This fits with the pattern used in other sections on this page but happy to rework if you think it could be clearer?

Reply:
Just looks a bit crazy to me.  `isPathfinderUser` will **always** be true in the `Pathfinder` function so it is redundant code and the tests are testing a scenario that will *never* happen.
Similarly testing `pathfinderUrl` here is silly as what needs to happen is for the full `div` including `h3` not to be shown - maybe sounds like the `<Pathfinder` should be moved to the top and logic moved into that component instead so can be tested on its own?

Reply:
You're right, of course. It appears that a number of conditional sections within quick look, including this one, are unit-tested based on conditions that have already been excluded in the main section, so I'll rework this into a self-contained function that either will or won't render the Pathfinder section, and can be driven via a unit test.